components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -53,6 +55,16 @@
       ],
       "run_at": "document_start",
       "all_frames": true
+    },
+    {
+      "matches": [
+        "*://*/*"
+      ],
+      "run_at": "document_start",
+      "js": [
+        "bundles/content-script.bundle.js",
+        "bundles/web-discovery-content-script.bundle.js"
```

Comment:
@keur I think we only need `bundles/web-discovery-content-script.bundle.js` here?